dvc/updater.py

```diff
@@ -25,7 +25,9 @@ class Updater(object):  # pragma: no cover
     def __init__(self, dvc_dir):
         self.dvc_dir = dvc_dir
         self.updater_file = os.path.join(dvc_dir, self.UPDATER_FILE)
-        self.lock = Lock(dvc_dir, self.updater_file + ".lock")
+        self.lock = Lock(
```

Comment:
You don't set lifetime here, is there some default value?

Reply:
Yes, 5 seconds. Plus 10 seconds, which flufl uses internally to defend against time desync.

Reply:
That is not acceptable, I can see it taking that long for the updater on bad network. Just set the longest time by default in Lock and don't configure it unnecessarily each time you use Lock. It is the way our old locks worked, it should stay that way.

Reply:
@Suor ?